### PR TITLE
Updates maintenance/license year to 2023

### PR DESCRIPTION
**Pull request by lmagyar**
# Proposed Changes

Documentation fix

### CODE REVIEW

**Comment by lmagyar** [comment]
Not stale, waiting for review/merge...

**Comment by lmagyar** [comment]
Not stale, waiting for review/merge...

File: tailscale/DOCS.md [hunk 1]
```diff
@@ -30,10 +30,12 @@ however, it is nice to know where you need to go later on.
...
 1. Check the logs of the "Tailscale" add-on to see if everything went well.
-1. Open the Web UI of the "Tailscale" add-on to complete authentication and
+1. Open the **Web UI** of the "Tailscale" add-on to complete authentication and
    couple your Home Assistant instance with your Tailscale account.
-   **Note:** Some browsers don't work with this step. It is recommended to
-   complete this step on a desktop or laptop computer using the Chrome browser.
+
+   **Note:** _Some browsers don't work with this step. It is recommended to
+   complete this step on a desktop or laptop computer using the Chrome browser._
+
```
Comment (frenck) [inline tailscale/DOCS.md:38]: Unrelated changes to this PR

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Setting review status correctly

**Comment by frenck** [comment]
superseded by #155

**Pull request closed without merging**